app/views/about/index.html.erb

```diff
@@ -29,11 +29,11 @@
               </thead>
               <tbody>
               <% @smart_proxies.each do |proxy| %>
-                <tr>
+                <tr class="proxy-show"  data-url="<%= ping_smart_proxy_path(proxy) %>">
                   <td><%= link_to_if_authorized proxy.name, hash_for_edit_smart_proxy_path(proxy) %></td>
```

Comment:
Now that there's a `show` page, I think it'd be more appropriate to link to the smart_proxy page.